testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/HiveProductTest.java

```diff
@@ -75,4 +81,13 @@ protected boolean isHiveVersionBefore12()
         return getHiveVersionMajor() == 0
                 || (getHiveVersionMajor() == 1 && getHiveVersionMinor() < 2);
     }
+
+    protected static void saveOnHdfs(HdfsClient hdfsClient, Path path, String location)
```

Comment:
Can you inject
```
HdfsClient hdfsClient
```
within `HiveProductTest`?

nit: In follow up PR you could make `TestAvroSchemaUrl` us this new method